platform/barefoot/docker-syncd-bfn-rpc.mk

```diff
@@ -19,7 +19,7 @@ endif
 $(DOCKER_SYNCD_BFN_RPC)_CONTAINER_NAME = syncd
 $(DOCKER_SYNCD_BFN_RPC)_VERSION = 1.0.0-rpc
 $(DOCKER_SYNCD_BFN_RPC)_PACKAGE_NAME = syncd
-$(DOCKER_SYNCD_BFN_RPC)_RUN_OPT += --net=host --privileged -t
```

Comment:
Could you share more insight why "--net=host is specified twice" ? Maybe we should fix the other place.

Reply:
Commit c948379 made some changes in [files/build_templates/docker_image_ctl.j2](https://github.com/Azure/sonic-buildimage/commit/c9483796dc7dc6ad68b19e3970d851ed43ffbad3#diff-130e5ab75471398db06b73a4c6e4b56517f42e3c1f17870b5309620fc0e2fafb). Specifically, it added some code here to explicitly specify which network namespace/topology the container should run in (with the `--net=` flag): either the host namespace (so that it has all of the host network interfaces accessible within the container), connected to the Docker bridge with veth interfaces (this is the default), or the namespace of another container.

If the container isn't a per-ASIC container (meaning there's only one for the system), then it'll put it in the host namespace. Otherwise, if it's in a multi-ASIC environment and the database container is being started, then that container will be connected via the default Docker bridge. Otherwise, it'll use the namespace of the database container for that ASIC.

This change also removed all of the explicit `--net=host` flags specified in the run options for other containers (see the files changed in `rules/` directory in that commit), but didn't handle the ones that were defined in the `platform/` directory. So I'm replicating the change made in that commit for these other containers.